.circleci/config.yml

```diff
@@ -414,6 +414,26 @@ jobs:
       - checkout
       - run: *run_docs_pragma_min_version
 
+  t_pyscripts_ubu:
+    docker:
+      - image: << pipeline.parameters.ubuntu-2004-docker-image >>
+    steps:
+      - checkout
+      - run:
+          name: Python unit tests
+          command: python3 test/pyscriptTests.py
```

Comment:
Is `python3` for sure installed?

Reply:
It runs in the CI :) So, yes?

Reply:
it's ubuntu so, you never know...

Reply:
Yeah. It's actually the plain `python` executable that is not :) Initially I used `python` and I was getting errors saying that it does not exist.

On Ubuntu the `python` symlink is a part of some package other than `python` (it's in `python-pip` I think?) so if you just install Python 3, it's not there.